shared/src/sources/balancer/swap.rs

```diff
@@ -123,26 +112,15 @@ impl WeightedPoolRef<'_> {
         .map(|bfp| out_reserves.downscale(bfp))
         .flatten()
     }
-
-    fn checked_get_amount_out(
-        &self,
-        out_token: H160,
-        (in_amount, in_token): (U256, H160),
-    ) -> Option<U256> {
-        let out_amount = self.unchecked_get_amount_out(out_token, (in_amount, in_token))?;
-        // We double check that resulting amount out can symmetrically provide an amount in.
-        self.unchecked_get_amount_in(in_token, (out_amount, out_token))?;
-        Some(out_amount)
-    }
```

Comment:
I see that these were removed in favour of `get_amount_*_`, but it doesn't appear they were actually re-implemented somewhere else... what happened with `unchecked_get_amount_*` is it even used anymore? Based on the changes here, I can only assume that we already had `get_amount_out_` and it was implemented identical to this method here that was removed.